model/src/main/kotlin/Repository.kt

```diff
@@ -48,4 +48,17 @@ data class Repository(
          * The configuration of the repository, parsed from the ".ort.yml" file.
          */
         val config: RepositoryConfiguration
-)
+) {
+    /**
+     * Return the path of [vcs] relative to [Repository.vcs], or null if [vcs] is neither [Repository.vcs] nor contained
+     * in [nestedRepositories].
+     */
+    fun getVcsPath(vcs: VcsInfo): String? {
+        fun VcsInfo.similar(other: VcsInfo) =
```

Comment:
I'm not so happy about the name "similar" here. Similar in what regard? Maybe something like "isCheckoutOf"?

Reply:
Renamed it to `matches` instead which is already used in other places, e.g. in excludes.